net-misc/usbip/usbip-4.13-r1.ebuild

```diff
@@ -24,9 +23,10 @@ RDEPEND=">=dev-libs/glib-2.6
 DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
-DOCS="AUTHORS README"
+S="${WORKDIR}/linux-${PV}/tools/usb/${PN}"
 
-S=${WORKDIR}/linux-${PV}/tools/usb/${PN}
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
+DOCS=( AUTHORS README )
```

Comment:
same, check if we need to define these explicitly

Reply:
Removed DOCS